### PR TITLE
C2C-18

### PR DESCRIPTION
**Pull request by reagan-meant**
Cleaned up forms.
Changes effected.
https://mekomsolutions.atlassian.net/browse/C2C-40
https://mekomsolutions.atlassian.net/browse/C2C-32
https://mekomsolutions.atlassian.net/browse/C2C-37
https://mekomsolutions.atlassian.net/browse/C2C-42
https://mekomsolutions.atlassian.net/browse/C2C-41
Also changed the triage form to have 3 options

### CODE REVIEW

File: configuration/concepts/nutrition.csv [hunk 1]
```diff
@@ -4,7 +4,7 @@ edaef9f4-2b5b-4b71-9019-74f1d40ad4d7,,CIEL:460,Oedema,Œdème,Oedema,Œdème,,,M
 8dfc3ec9-771a-42b6-ad45-aa0ac15b660c,,,MUAC<115mm,PB<115mm,MUAC<115mm,PB<115mm,,,Misc,N/A,,,,,,,,,,,
 6a7ea608-0546-40f1-92d2-cd020e980414,,CIEL:1655,Admission Criteria,Critère d'admission,Admission Criteria,Critère d'admission,,,Misc,Coded,,Oedema;Weight-for-height<-3SD/BMI<16;MUAC<115mm,,,,,,,,,
 a2224476-1827-4270-9ace-c8fd7898fd21,,,Readmission,Réadmission,Readmission,Réadmission,,,Misc,Coded,,Yes;No,,,,,,,,,
-fc1f3f86-9332-4714-a4f4-ce88c173a5c2,,CIEL:163102,Weight goal,Objectif de poids,Weight goal,Objectif de poids,,,Misc,Numeric,,,,TRUE,Kg,,,,,,
+fc1f3f86-9332-4714-a4f4-ce88c173a5c2,,CIEL:163102,Weight Goal,Objectif de poids,Weight Goal,Objectif de poids,,,Misc,Numeric,,,,TRUE,Kg,,,,,,
```
Comment (rbuisson) [inline configuration/concepts/nutrition.csv:7]: It seems that the decimals are allowed (form can be saved), but the UI suggests it's not:
Screenshot 2020-10-22 at 20 15 39
Do you experience the same thing on your browser? This is also the case for Weight in Vital Signs form.

**Review by rbuisson — CHANGES_REQUESTED** [review]
Looks good. Just that possible issue with the Weight Goal with decimals but please confirm first that you have the issue too. And if so, that's likely going to be resolved in another ticket.

Another change (from a change of requirements):
Could you change the "number of days" in past admissions to be done the usual way as "Number" of "Days/Weeks/Months"?

File: configuration/concepts/nutrition.csv (hunk 1 above)
Comment (reagan-meant) [reply configuration/concepts/nutrition.csv:7]: Everything works out well on my end

Comment (reagan-meant) [reply configuration/concepts/nutrition.csv:7]: Fixed Historique de sante form too

Comment (rbuisson) [reply configuration/concepts/nutrition.csv:7]: OK then. Thanks

**Review by rbuisson — APPROVED** [review]

**Pull request merged**